components/js/ops-overview.js

```diff
@@ -1,470 +1,557 @@
+    data_transform: function(x) {
+      return [x.length];
     },
-    {
-        'id' : 'bcycle-trips',
-        'row_container_id' : 'panel-row-2',
-        'display_name' : '  B-Cycle Trips',
-        'icon' : 'bicycle',
-        'init_val' : 0,
-        'format' : 'thousands',
-        'infoStat' : true,
-        'resource_id' : 'cwi3-ckqi',
-        'caption' : '# of Austin B-Cycle trips taken this year.',
-        'query' : function(){
-            var y = new Date().getFullYear().toString();
-            return 'select count(checkout_date) as count where date_extract_y(checkout_date)=' + y;
-        }(),
-        'data_transform' : function(x) { return [x[0]['count']] },
-        'update_event' : 'bcycle_trip_pub'
-    }
+    update_event: "knack_data_pub_detectors_knack_socrata"
+  },
+  // {
+  //     'id' : 'school-beacons',
+  //     'row_container_id' : 'panel-row-1',
+  //     'display_name' : 'School Beacons',
+  //     'icon' : 'bus',
+  //     'init_val' : 0,
+  //     'format' : 'round',
+  //     'data' : [537],
+  //     'infoStat' : true,
+  //     'caption' : '',
+  //     'update_event' : undefined
+  // }
+  {
+    id: "work_orders_signals",
+    row_container_id: "panel-row-2",
+    display_name: "Signal Work Orders",
+    icon: "wrench",
+    init_val: 0,
+    format: "thousands",
+    infoStat: true,
+    caption: "Work orders completed at traffic signal assets",
+    query:
+      "select count(created_date) as count where upper(work_order_status) in ('SUBMITTED', 'CLOSED') and fiscal_year=" +
+      fiscal_year,
+    resource_id: "v2ig-5yw3",
+    data_transform: function(x) {
+      return [x[0]["count"]];
+    },
+    update_event: "knack_data_pub_work_orders_signals_knack_socrata"
+  },
+  {
+    id: "bcycle-trips",
+    row_container_id: "panel-row-2",
+    display_name: "  B-Cycle Trips",
+    icon: "bicycle",
+    init_val: 0,
+    format: "thousands",
+    infoStat: true,
+    resource_id: "cwi3-ckqi",
+    caption: "# of Austin B-Cycle trips taken this year.",
+    query: (function() {
+      var y = new Date().getFullYear().toString();
+      return (
+        "select count(checkout_date) as count where date_extract_y(checkout_date)=" +
+        y
+      );
+    })(),
+    data_transform: function(x) {
+      return [x[0]["count"]];
+    },
+    update_event: "bcycle_trip_pub"
+  },
+  {
+    id: "dockless-trips-total-count",
+    row_container_id: "panel-row-4",
+    display_name: "  Total Trips (scooter & bike)",
+    icon: "mobile",
+    init_val: 0,
+    format: "thousands",
+    infoStat: true,
+    resource_id: "7d8e-dm7r",
+    caption: "# of total Dockless Mobility trips taken.",
+    query: (function() {
+      return "select count(id) as total_trips where trip_distance * 0.000621371 >= 0.1 and trip_distance * 0.000621371 < 500 and trip_duration < 86400";
+    })(),
+    data_transform: function(x) {
+      return [x[0]["total_trips"]];
+    },
+    update_event: "dockless_trips"
```

Comment:
assumed this was the right value for all for new cards by looking at the publisher logs, but I also noticed there are at least 3 other types of dockless publishing events.

<img width="146" alt="screen shot 2018-12-11 at 4 17 02 pm" src="https://user-images.githubusercontent.com/5697474/49833822-4ae4da80-fd60-11e8-922d-5f6b9d92731b.png">